src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs

```diff
@@ -51,12 +56,12 @@ public void Arrange()
                 UkPrn = ExpectedUkPrn
             };
 
-            _providerPermissionsService = new Mock<IProviderPermissionsService>();
+            _providerService = new Mock<IProviderService>();
             _validator = new Mock<IValidator<GetTrustedEmployersQuery>>();
 
-            _handler = new GetTrustedEmployersQueryHandler(_providerPermissionsService.Object, _validator.Object);
+            _handler = new GetTrustedEmployersQueryHandler(_providerService.Object, _validator.Object);
```

Comment:
GetTrustedEmployersQueryHandler => GetTrustedAccountLegalEntitiesQueryHandler
plus all other files/classes - request, response, test files, validator..

Reply:
I think I would be tempted to keep the name of this as it does describe the action that is occurring.